mockingbird/src/commonMain/kotlin/com/careem/mockingbird/test/Capture.kt

```diff
@@ -14,6 +14,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+@file:Suppress("NON_EXHAUSTIVE_WHEN")
```

Comment:
Can we make the warning suppression a bit more granular please?

Reply:
Can we remove it and make the when exhaustive instead? I think it complains about `null` not having a behavior, I think we can throw exception in this case